src/deck/guides/konnect.md

```diff
@@ -0,0 +1,227 @@
+
+Similar to Gateway workspaces, decK can only target one runtime group at a time.
+Managing multiple runtime groups requires a separate state file per group.
+
+You _cannot_ use decK to publish content to the Dev Portal, manage application registration, or configure custom plugins.
+
+## {{site.konnect_short_name}} flags
+
+You can use any regular `deck` commands (such as `ping`, `diff`, or `sync`) with `--konnect` flags to target {{site.konnect_short_name}} instead of a self-managed {{site.base_gateway}} control plane.
+
+You can also [save {{site.konnect_short_name}} credentials to a decK config file](#authenticate-with-a-deck-config-file) to target {{site.konnect_short_name}} by default.
+
+If you don't pass any {{site.konnect_short_name}} parameters to decK, decK looks for a local {{site.base_gateway}} instance instead.
+
+{:.note}
+> Prior to decK 1.12, decK provided `deck konnect` commands.
+These commands are deprecated and have been replaced with flags.
+
+`--konnect-addr`
+:  Address of the {{site.konnect_short_name}} endpoint. (Default: `"https://us.api.konghq.com"`)
+
+`--konnect-email`
+:  Email address associated with your {{site.konnect_short_name}} account.
+
+`--konnect-password`
+:  Password associated with your {{site.konnect_short_name}} account.
+This takes precedence over the `--konnect-password-file` flag.
+
+`--konnect-password-file`
+:  File containing the password to your {{site.konnect_short_name}} account.
+
+`--konnect-runtime-group-name`
+:  {{site.konnect_short_name}} runtime group name.
+
+## Authenticate with {{site.konnect_short_name}}
+
+decK looks for {{site.konnect_short_name}} credentials in the following order of precedence:
+
+1. Password set with the `--konnect-password` flag
+2. decK configuration file (default: `~/.deck.yaml`)
+3. File passed with the `--konnect-password-file` flag
+
+For example, if you have both a decK config file and a {{site.konnect_short_name}} password file, decK uses the password in the config file.
+
+### Authenticate with CLI flags
+
+There are two ways to authenticate through CLI flags: by passing the password in `--konnect-password` directly, or by saving it to a file and passing the filename to decK with `--konnect-password-file`.
+
+With `konnect-password`:
+
+```sh
+deck ping \
+  --konnect-email YOUR_EMAIL \
+  --konnect-password YOUR_PASSWORD
+```
+
+Or, save your {{site.konnect_short_name}} password to a file, then pass the file using the `--konnect-password-file` flag:
+
+```sh
+deck ping \
+  --konnect-email YOUR_EMAIL \
+  --konnect-password-file /PATH/TO/FILE
+```
+
+### Authenticate with a decK config file
+
+The default decK configuration file is `~/.deck.yaml`.
+You can store {{site.konnect_short_name}} credentials in this file to pass them more securely.
+For example:
+
+```
+konnect-email: example@email.com
+konnect-password: YOUR_PASSWORD
+```
+
+Alternatively, you can save your password in a separate file, then specify the password file instead of a password:
```

Comment:
```suggestion
Alternatively, you can save your password in a separate file, then specify the password file instead of a password. 
For example:
```
Just to mirror the one above.